### PR TITLE
Shortest distance

### PR DESCRIPTION
**Pull request by JarrettSJohnson**
Finds the shortest atom pair between two selections and creates a distance object between the two closest.

First tried using chempy's `Atom` class, but didn't seem to have an equivalent `model` for selection purposes.

### CODE REVIEW

File: psico/querying.py [hunk 1]
```diff
@@ -510,4 +512,104 @@ def extinction_coefficient(selection="all", state=-1, *, quiet=1, _self=cmd):
...
+    # Show the shortest distance as a dashed line
+    sele_a = closest_pair[0].asSelection()
+    sele_b = closest_pair[1].asSelection()
+    obj_name = _self.get_unused_name(obj_name)
+    _self.distance(obj_name, sele_a, sele_b, quiet=quiet)
+
+    min_distance = sqrt(min_distance_sq)
+    if not quiet:
+        print(
+            f"Shortest distance: {min_distance} Å between {sele_a} and {sele_b}")
```
Comment (speleo3) [inline psico/querying.py:611]: Rounding would be nice: `{min_distance:.2f}`

**Review by speleo3 — CHANGES_REQUESTED** [review]
Thanks Jarrett for this PR, looks great!

**Comment by JarrettSJohnson** [comment]
Thanks for looking them over. Addressed your comments.

File: psico/querying.py [hunk 2]
```diff
@@ -488,6 +490,103 @@ def extinction_coefficient(selection="all", state=-1, *, quiet=1, _self=cmd):
...
+
+    sele_a = string: first atom selection
+
+    sele_b = string: second atom selection
+
+    sele_a_state = state of sele a {default: current state}
+
+    sele_b_state = state of sele b {default: current state}
+
+    obj_name = string: name of the object to create {default: shortest}
```
Comment (speleo3) [inline psico/querying.py:516]: can you fix the argument names in the docstring?

Comment (JarrettSJohnson) [reply psico/querying.py]: Done. Sorry bout that.

**Pull request merged**